webpack.test.bootstrap.ts

```diff
@@ -0,0 +1,2 @@
+let testContext = (<{ context?: Function }>require).context('./', true, /\.test\.ts/);
```

Comment:
Yeah, she told me. I wonder if we shouldn't be able to modify this and get rid of our dependency on browserify. See [this comment](https://github.com/rangle/batarangle/pull/72#issuecomment-159345392) and [this one](https://github.com/rangle/batarangle/pull/72#issuecomment-159350027). Essentially...I'm wondering if there's a module system polyfill (even es6-loader) that we need to include in the test webpack.